packages/core/src/codewhisperer/models/constants.ts

```diff
@@ -451,37 +454,47 @@ export const unsupportedJavaVersionChatMessage =
     'Sorry, currently I can only upgrade Java 8 or Java 11 projects. For more information, see the [Amazon Q documentation](https://docs.aws.amazon.com/amazonq/latest/aws-builder-use-ug/code-transformation.html#prerequisites).'
 
 export const failedToStartJobChatMessage =
-    "Sorry, I couldn't begin the transformation. Please try starting the transformation again."
+    'Sorry, I could not begin the transformation. Please try starting the transformation again.'
 
 export const failedToStartJobNotification =
-    "Amazon Q couldn't begin the transformation. Please try starting the transformation again."
+    'Amazon Q could not begin the transformation. Please try starting the transformation again.'
 
 export const failedToStartJobTooManyJobsChatMessage =
     'Sorry, I couldn’t begin the transformation. You have too many active transformations running. Please try again after your other transformations have completed.'
 
 export const failedToStartJobTooManyJobsNotification =
-    "Amazon Q couldn't begin the transformation. You have too many active transformations running. Please try again after your other transformations have completed."
+    'Amazon Q could not begin the transformation. You have too many active transformations running. Please try again after your other transformations have completed.'
+
+export const failedToStartJobLinesLimitNotification =
+    'Amazon Q could not begin the transformation. Your code exceeds the lines of code limit. Please try again with a smaller project.'
```

Comment:
There are currently two scenarios for failedToStartJobLinesLimitNotification:
1. "Job Lines of Code limit breached" - single project exceeded line limit.
2. "Monthly aggregated Lines of Code limit breached" - completed jobs in the current month exceeded aggregated line limit across jobs.

This message always assume case 1.